.mergify.yml

```diff
@@ -4,23 +4,35 @@ pull_request_rules:
       - author~=dependabot\[bot\]|dependabot-preview\[bot\]
       - status-success=Travis CI - Pull Request
       - status-success=continuous-integration.azure-devops
+      - or:
+          - -title~=(?i)pyinstaller
```

Comment:
The `-` is so hard to see. Can we make the condition more obvious using the `not:` condition?

Reply:
Also, do we need to match `(?i).*pyinstaller.*`?